07-mongodb-mongoose/02-rest-api/controllers/products.js

```diff
@@ -0,0 +1,35 @@
+const Product = require('../models/Product');
+
+module.exports.productsBySubcategory = async function productsBySubcategory(ctx, next) {
+  try {
+    const {subcategory} = ctx.request.query;
+    const products = await Product.find(subcategory ? {subcategory} : {});
```

Comment:
Can I combine `subcategory ? {subcategory} : {}` somehow?

Reply:
What do you mean by "combine"?